app/components/cache-classification.js

```diff
@@ -0,0 +1,67 @@
+import apiClient from 'panoptes-client/lib/api-client';
+
+class CacheClassification {
+  constructor() {
+    this.state = {
+      cachedClassification: null,
+    };
+
+    window.cachedClassification = this.state.cachedClassification;
```

Comment:
per conversation with Sarah, we're thinking about moving this to classifier/index.cjsx